docs/azure_arc_jumpstart/azure_arc_k8s/aks_stack_hci/aks_hci_powershell/_index.md

```diff
@@ -0,0 +1,139 @@
+
+![Screenshot showing the AKS on HCI deployment script](./01.png)
+
+* To run the script open PowerShell as an administrator, navigate to the [script folder](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/aks_stack_hci/powershell/) and run: 
+
+  ```powershell
+  .\aks_hci_deploy.ps1
+  ```
+![Screenshot showing the script input](./02.png)
+
+* The script will prompt to provide a set of parameters to configure your cluster: 
+    - **imageDir:** path to the directory where AKS on Azure Stack HCI will store its VHD images, provide a shared path or SMB for multinode
+    - **cloudConfigLocation:** path to the directory where the cloud agent will store its configuration, provide a shared path or SMB for multinode
+    - **vnetName:** the name of the virtual switch to connect the virtual machines to. If you already have an external switch on the host, you should pass the name of the switch here. 
+    - **clusterName:** a name for your AKS cluster, must be lowercase. 
+    - **controlPlaneNodeCount:** number of nodes for your control plane, should be an odd number 1, 3 or 5.
+    - **linuxNodeCount:** number of Linux node VMs for your cluster, if you do not need Linux nodes input 0.
+    - **windowsNodeCount:** number of Windows node VMs for your cluster, if you do not need Windows nodes input 0.
+
+    > **Note: the script may take upto 30 minutes to run**
+
+* Once the script has completed its run, you will see an output as follows: 
+
+![Screenshot showing the script output](./03.png)
+
+* To be able to connect to the Azure Kubernetes Service cluster run the command: 
+
+  ```powershell
+  Get-AksHciCredential -clusterName <cluster_name>
+  ```
+![Screenshot showing the output retrieving credentials](./04.png)
+
+## Connecting to Azure Arc
+
+* Now that you have a running AKS cluster, edit the environment variables section in the included [az_connect_aks](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/aks_stack_hci/powershell/aks_hci_connect.ps1) shell script.
+
+![Screenshot showing environment variables aks_hci_connect.ps1](./05.png)
+
+* After editing the environment variables in the aks_hci_connect PowerShell script to match your parameters, save the file and then run it using the .\aks_hci_connect.ps1 command.
+
+* Once the script run has finished, the AKS cluster on HCI will be projected as a new Azure Arc cluster resource.
+
+![Screenshot showing environment variables aks_hci_connect.ps1](./06.png)
```

Comment:
Before showing the cluster resource, have a screenshot showing the cluster as part of the resource group